lib/matplotlib/axes/_axes.py

```diff
@@ -2339,8 +2339,16 @@ def broken_barh(self, xranges, yrange, **kwargs):
         .. plot:: mpl_examples/pylab_examples/broken_barh.py
         """
         # process the unit information
-        self._process_unit_info(xdata=xranges[0],
-                                ydata=yrange[0],
+        if len(xranges):
+            xdata = six.next(iter(xranges))
```

Comment:
I think we can just use `next` here https://docs.python.org/2/library/functions.html#next  I think `six` provides it to support back to 2.4 or something like that.